pom.xml

```diff
@@ -139,6 +139,78 @@
         <module>optaplanner-distribution</module>
       </modules>
     </profile>
+
+    <!-- Disable Java 8 active by default profiles from JBoss -->
```

Comment:
Let's add a expiration condition comment, at which point all these extra profiles can be removed.

Reply:
Please create an issue upstream on jboss-parent if that hasn't been done yet, and link it here.